code/game/objects/effects/spiderwebs.dm

```diff
@@ -101,6 +101,38 @@
 	else if(isprojectile(mover))
 		return prob(30)
 
+/obj/structure/spider/solid
+	name = "solid web"
+	icon = 'icons/effects/effects.dmi'
+	desc = "A solid thick wall of web, airtight enough to block air flow."
```

Comment:
```suggestion
	desc = "A solid wall of web, thick enough to block air flow."
```
Nitpick but you don't need both of these words here, they basically mean the same thing.
And same with "airtight enough to block airflow", that is already what airtight means.